ogc/tiles/templates/tiles.go.html

```diff
@@ -1,61 +1,124 @@
-                    <td><a href="https://www.opengis.net/def/crs/EPSG/0/3857">EPSG:3857</a></td>
-                    <td>vector</td>
-                </tr>
-                {{end}}
-                {{end}}
-            </tbody>
-        </table>
+      <p>
+        Deze hele dataset is beschikbaar via OGC API Tiles. Tiles kunnen zowel vector als raster data
+        bevatten en in meerdere projecties beschikbaar worden gesteld. In het geval van vector tiles
+        biedt PDOK ook styles aan.
+      </p>
     </div>
-</div>
-{{end}}
+  </div>
+  <div class="row">
+      {{ $baseUrl := .Config.BaseURL }}
+      {{ $defaultSrs := (index .Config.OgcAPI.Tiles.SupportedSrs 0)}}
+      {{ $projections := dict "EPSG:28992" "NetherlandsRDNewQuad" "EPSG:4258" "EuropeanETRS89_GRS80Quad_Draft" "EPSG:3857" "WebMercatorQuad" }}
+    <div class="col-md-4">
+      <table class="table">
+        <tbody>
+        <tr>
+          <th>Tile Matrix Set</th>
+          <td>
+              {{ if eq (len .Config.OgcAPI.Tiles.SupportedSrs) 1 }}
+                  {{ get $projections $defaultSrs.Srs }}
+              {{ else }}
+                <select id="srs">
+                    {{ range $srs := .Config.OgcAPI.Tiles.SupportedSrs }}
+                      <option value="{{ $srs.Srs }}">{{ get $projections (index $srs).Srs }}</option>
+                    {{ end }}
+                </select>
+              {{ end }}
+          </td>
+        </tr>
+        <tr>
+          <th>Schema</th>
+          <td id="field-schema"><a id="href-schema"
+                                   href="https://www.opengis.net/def/crs/EPSG/0/{{ trimPrefix "EPSG:" $defaultSrs.Srs }}">HTML
+              (link naar externe website)</a></td>
+        </tr>
+        <tr>
+          <th>Bijgewerkt door PDOK</th>
+          <td id="field-updated">{{ if .Config.LastUpdated}}{{ toDate "2006-01-02T15:04:05Z07:00" .Config.LastUpdated | date "2006-01-02" }}{{ end }}</td>
+        </tr>
+        <tr>
+          <th>Type</th>
+          <td id="field-type">{{ (index .Config.OgcAPI.Tiles.Types 0) }}</td>
+        </tr>
+        <tr>
+          <th>CRS</th>
+          <td id="field-srs">{{ $defaultSrs.Srs }}</td>
+        </tr>
+        <tr>
+          <th>Metadata Tiles</th>
+          <td id="field-metadata"><a id="href-metadata"
+                                     href="tiles/{{ get $projections $defaultSrs.Srs }}">Link</a>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="col-md-8">
+      <app-vectortile-view id="tile-vectortile-view" style="width: 800px; height: 600px;"
+                           tile-url="{{ $baseUrl }}/tiles/{{ get $projections $defaultSrs.Srs }}"
+                           {{ if .Config.OgcAPI.Styles }}style-url="{{ $baseUrl }}/styles/{{ .Config.OgcAPI.Styles.Default }}"{{ end }}
+                           zoom="{{ $defaultSrs.ZoomLevelRange.End }}"
+                           center-x="5.3896944"
```

Comment:
Gaat dit met die coordinaten nu goed als je switched tussen projecties? Lijken WebMercator coordinaten namelijk, anders even met John schakelen?

Reply:
ik zal het navragen

Reply:
Ja Het component heeft als parameters altijd WebMercator.  Ook voor b.v de RD projectie. Zie voorbeelden in index.html.  (d.m.v. useGeographic() in Openlayers)